src/Core/src/Platform/Android/KeyboardExtensions.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.Text;
+
+namespace Microsoft.Maui.Platform.Android
+{
+	public static class KeyboardExtensions
+	{
+		public static InputTypes ToInputType(this Keyboard self)
```

Comment:
If this is the same as the one in Compatibility, maybe we should move that into here and delete that one? No use having duplicate logic.